plugins/blend/README.md

```diff
@@ -0,0 +1,31 @@
+# blend
+
+This plugin adds [mix-blend-mode](https://developer.mozilla.org/en-US/docs/Web/CSS/mix-blend-mode) utilities to Tailwind.
```

Comment:
@gregkohn Can you look over the blend docs and the plugin itself since I think you wrote it originally? Also let me know if you have a more recent version of it.